conda/meta.yaml

```diff
@@ -20,9 +20,9 @@ test:
     - ts-salobj {{ salobj_version }}
     - ts-scriptqueue
     - ts-observatory-model
-    - ts-astrosky-model
+    - ts-astrosky-model >1.4
     - ts-dateloc
-    - rubin-sim =0
+    - rubin-sim =1
```

Comment:
In ts_astrosky_model you put `>=1`. Why is `=1` here sufficient? Same question for line 57. Also, same question as for ts_astrosky_model: shouldn't more dependencies be mentioned in the `requirements` -> `build` section?

Reply:
ah! good catch in the rubin-sim dependency.. I'll update it here... thanks